### PR TITLE
docs: change properties with function value to methods

### PR DESCRIPTION
**Pull request by fhinkel**
Internally, see b/206943288

### CODE REVIEW

File: src/bucket.ts [hunk 1]
```diff
@@ -629,7 +629,10 @@ class Bucket extends ServiceObject {
   acl: Acl;
   iam: Iam;
 
-  getFilesStream: (query?: GetFilesOptions) => Readable;
+  getFilesStream(query?: GetFilesOptions): Readable {
+    // placeholder body, overwritten in constructor
```
Comment (bcoe) [inline src/bucket.ts:633]: This seems reasonble to me, but I would open a ticket in parser to fix this issue, and reference the ticket # here.

Comment (fhinkel) [reply src/bucket.ts:633]: You mean in api-documnter? Yeah, I can open an issue there.

**Review by bcoe — APPROVED** [review]

**Review by shaffeeullah — CHANGES_REQUESTED** [review]
Does this PR have to be `fix`? Can it be `docs`?

**Comment by fhinkel** [comment]
> Does this PR have to be `fix`? Can it be `docs`?

Sure!

**Review by shaffeeullah — APPROVED** [review]

File: src/bucket.ts (hunk 1 above)
Comment (fhinkel) [reply src/bucket.ts:633]: done: https://github.com/microsoft/rushstack/issues/3031

**Pull request merged**